docs/process.md

```diff
@@ -41,105 +42,65 @@ Costs are
+
+During this "Pre-RFC" phase, your proposal isn't in the RFC tracker.  It's not an RFC yet!
+
+During this phase, the proposer is responsible for moving things forward.  If
+they contact the PSC for approval to file a draft, and the PSC does not
+respond, it's the proposer who should be keeping track of that.
+
+### Draft Proposal
+
+The PSC has agreed that you should write a formal draft proposal.  You get the template document and fill it out.  You take its advice, thinking hard about what goes in each section.  Then you post it to p5p as an email with the subject "PROPOSAL:  my great idea".  Members of the list will reply with more questions and suggested amendments.  You should read them and amend the proposal to clarify your ideas or react to valid criticism.
+
+During this phase, you (the proposer) are responsible for moving things
+forward.
+
+When you think your idea has been sufficiently scrutinized, and you gotten all the feedback you're going to benefit from, submit your updated document to the PSC.  They will either request more discussion, reject the proposal, or enter it into the tracker with the status **Exploratory**.
+
+### Exploratory Status
+
+When a formal draft has been discussed sufficiently, submitted to the PSC, and is under consideration, it is entered into the proposal tracker, with the status **Exploratory**.  The PSC (or other deputized folk) will begin the process of vetting the idea.  They will review discussion about the idea, they will produce a list of questions not yet answered, and they will press existing core team members (or other experts) for input.
+
+The PSC (or deputies) will eventually either:
+ * move the document to **Implementing** status (see below) because they believe it is ready for implementation
+ * move the document to **Rejected** status because they believe it should not be implemented; This may come with advice on how to formulate an alternate proposal that has more chance of being accepted.  This isn't used for "it needs some edits", but for "it's fundamentally deeply flawed."
+ * move the document to the **Expired** status because the original proposer (or other vital personnel) are not responsive
+
+During this phase, the PSC is responsible for moving the proposal forward.
+
+### Implementing
+
+When a proposal is accepted, the PSC is stating that they'd like to see the idea implemented, and that they are likely to merge an implementation if it doesn't bring with it any unwelcome surprises or complications.
+
+If no implementation has made progress for three months, the document moves to **Expired**.
```

Comment:
There ought to be some sort of resurrection process, yes. Maybe via a "is this still a good idea?" test, depending how long it has been since it expired.